src/Altinn.Notifications.Core/Services/OrderRequestService.cs

```diff
@@ -10,33 +10,35 @@
 namespace Altinn.Notifications.Core.Services;
 
 /// <summary>
-/// Implementation of the <see cref="IEmailNotificationOrderService"/>. 
+/// Implementation of the <see cref="IOrderRequestService"/>. 
 /// </summary>
-public class EmailNotificationOrderService : IEmailNotificationOrderService
+public class OrderRequestService : IOrderRequestService
 {
     private readonly IOrderRepository _repository;
     private readonly IGuidService _guid;
     private readonly IDateTimeService _dateTime;
-    private readonly string _defaultFromAddress;
+    private readonly string _defaultEmailFromAddress;
+    private readonly string _defaultSmsSender;
 
     /// <summary>
-    /// Initializes a new instance of the <see cref="EmailNotificationOrderService"/> class.
+    /// Initializes a new instance of the <see cref="OrderRequestService"/> class.
     /// </summary>
-    public EmailNotificationOrderService(IOrderRepository repository, IGuidService guid, IDateTimeService dateTime, IOptions<NotificationOrderConfig> config)
+    public OrderRequestService(IOrderRepository repository, IGuidService guid, IDateTimeService dateTime, IOptions<NotificationOrderConfig> config)
     {
         _repository = repository;
         _guid = guid;
         _dateTime = dateTime;
-        _defaultFromAddress = config.Value.DefaultEmailFromAddress;
+        _defaultEmailFromAddress = config.Value.DefaultEmailFromAddress;
+        _defaultSmsSender = config.Value.DefaultSmsSender;
     }
 
     /// <inheritdoc/>
-    public async Task<(NotificationOrder? Order, ServiceError? Error)> RegisterEmailNotificationOrder(NotificationOrderRequest orderRequest)
+    public async Task<(NotificationOrder? Order, ServiceError? Error)> RegisterNotificationOrder(NotificationOrderRequest orderRequest)
     {
         Guid orderId = _guid.NewGuid();
         DateTime created = _dateTime.UtcNow();
 
-        var templates = SetFromAddressIfNotDefined(orderRequest.Templates);
+        var templates = SetFromAddressOrSenderNumberIfNotDefined(orderRequest.Templates);
```

Comment:
To simplify the method name could we just call it: SetSenderIfNotDefined? This would encompass both from adress and sms sender.